### ISSUE TITLE
subclass `CommandTree`

### ISSUE THREAD

**Issue by deadaf** [quotientbot/Quotient-Bot#145]
With every `help` command we make an API call to get a list of all slash commands in the bot. A better way to do this might be overriding `fetch_commands` method to cache the results. 

Also, we need to figure out a way to mention sub-commands since parents mention doesn't work & is basically useless. 

![image](https://github.com/quotientbot/Quotient-Bot/assets/72350242/126982ff-ab45-408b-b103-ffa98b4c076d)

**Comment by rtk-rnjn** [comment]
monkey patching `fetch_commands()` is really bad idea, I say we fetch and cache app commands on bot login.
And create arbitrary function something like `Quotient.get_app_command()` similar to dpy `Bot.get_command()`

**Comment by deadaf** [comment]
yeah actually that too is a good idea & even less work tbh. I think a method in the `Quotient` should be a fine approach here. 
It will fetch commands for the first time then every time it is called, it will return cached results.
